docs/rest-api/intro/posting.md

```diff
@@ -0,0 +1,42 @@
+---
+slug: /rest-api/intro/posting
+sidebar_label: Posting
+---
+
+# Posting
+:::note
+This endpoint requires [authentication](/rest-api/intro/authentication).
+:::
+
+The endpoint for posting into home is `/home`, sending a POST request with this body:
```

Comment:
Could the post size restrictions and how posts can't only consist of whitespace be explained here?

Reply:
Sure.